.github/workflows/deploy-prod-updates.yml

```
@@ -57,4 +57,4 @@ jobs:
 
       - name: Sync Rumba update
         run: |-
-          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r rumba-updates/ gs://updates-prod-mdn/rumba-bcd-updates
+          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -r rumba-updates/ gs://updates-prod-mdn/rumba-bcd-updates
```

We can keep `-d`, even if the `aws s3 sync` command didn't have the `--delete` option.

Just checked and both the S3 and GCS bucket just have a single file `bcd-updates.json`, so all good.

```suggestion
          gsutil -m -h "Cache-Control:public, max-age=86400" rsync -d -r rumba-updates/ gs://updates-prod-mdn/rumba-bcd-updates
```